suite-common/message-system/config/config.v1.json

```diff
@@ -1332,18 +1332,18 @@
                 "variant": "info",
                 "category": ["banner"],
                 "content": {
-                    "en": "Testnet account has been updated to new Testnet 4.",
-                    "es": "Testnet account has been updated to new Testnet 4.",
-                    "cs": "Testnet account has been updated to new Testnet 4.",
-                    "ru": "Testnet account has been updated to new Testnet 4.",
-                    "ja": "Testnet account has been updated to new Testnet 4.",
-                    "hu": "Testnet account has been updated to new Testnet 4.",
-                    "it": "Testnet account has been updated to new Testnet 4.",
-                    "fr": "Testnet account has been updated to new Testnet 4.",
-                    "de": "Testnet account has been updated to new Testnet 4.",
-                    "tr": "Testnet account has been updated to new Testnet 4.",
-                    "pt": "Testnet account has been updated to new Testnet 4.",
-                    "uk": "Testnet account has been updated to new Testnet 4.."
+                    "en": "Bitcoin Testnet account has been updated to new Testnet4.",
+                    "es": "La cuenta de Bitcoin Testnet ha sido actualizada al nuevo Testnet4.",
+                    "cs": "Účet Bitcoin Testnet byl aktualizován na nový Testnet4.",
+                    "ru": "Аккаунт Bitcoin Testnet был обновлён до нового Testnet4.",
+                    "ja": "Bitcoinテストネットのアカウントは新しいTestnet4に更新されました。",
```

Comment:
Network name (Bitcoin Testnet) is never translated, so I think in every language it should be `Bitcoin Testnet`